src/registrar/utility/email.py

```diff
@@ -0,0 +1,34 @@
+"""Utilities for sending emails."""
+
+import boto3
+
+from django.conf import settings
+from django.template.loader import get_template
+
+
+def send_templated_email(template_name: str, to_address: str, context={}):
```

Comment:
**Non-blocking:** Thinking ahead to the future, how is this signature going to accommodate TXT + HTML messages and messages with To/Cc/Bcc?

It's fine as is, but if you want to improve it before it is used in many other places, it could save your future self some refactoring headache.

Reply:
Personally, I'm happy with us refactoring this in the future when we need more complicated messages.